src/main/java/com/lewisd/maven/lint/rules/AbstractReduntantVersionRule.java

```diff
@@ -47,20 +47,20 @@ protected void checkForRedundantVersions(final MavenProject mavenProject,
         }
 
         String version = resolveVersion(modelObject, resolvedModelObject);
-        String inheritedVersion = modelUtil.getVersion(tryResolveObject(inheritedObject));
+        String inheritedVersion = getModelUtil().getVersion(tryResolveObject(inheritedObject));

```

Comment:
You picked the point `99%`. Special cases make software complex. So does it hurt so much to handle the other 1% like the rest? So nobody needs space in its brain while changing your code. This is just risk minimization.

Keep also mind a common sense of coding helps onboard people into your project. ;)